main.js

```diff
@@ -0,0 +1,103 @@
+//// TOWERS OF HANOI - MODULE 1 PROJECT ////
+
+/// 1: Game Board Initilization ///
+  var TowersOfHanoi = {
+    GameBoard: [
+      [5, 4, 3, 2, 1], // PEG 1 //
+      [],              // PEG 2 //
+      []               // PEG 3 //
+    ]
+  }
+
+/// 1.5: Game Instructions ///
+  TowersOfHanoi.ShowInstructions = function () {
+    alert("Welcome to Towers of Hanoi!\n\nInstructions:\n- Move all discs from Peg 1 to either Peg 2 or Peg 3.\n- You can only move one disc at a time.\n- A larger disc cannot be placed on top of a smaller disc.\n- Use TowersOfHanoi.MoveDisc(fromPeg, toPeg) to make a move.\n\nExample: TowersOfHanoi.MoveDisc(1, 2) moves a disc from Peg 1 to Peg 2.\n\nGood luck!");
+  }
+  // Shows Instructions //
+    TowersOfHanoi.ShowInstructions();
+
+/// 2: Print Board ///
+  TowersOfHanoi.PrintBoard = function () {
+    console.clear();
+    this.GameBoard.map((peg, index) => console.log(`--- ${index + 1}: ${peg.join(" ")}`));
```

Comment:
wrong helper, use forEach. More performant. Map returns an array that you are not storing